performance/src/gatling/simulations/org/pmiops/databrowser/Scenarios.scala

```diff
@@ -0,0 +1,23 @@
+package org.pmiops.databrowser
+
+import io.gatling.core.Predef.scenario
+import io.gatling.core.structure.ScenarioBuilder
+
+sealed trait UserScenario { val name: String }
```

Comment:
Thanks for this code example, this makes things very concrete.

Now that I think I understand what Gatling does  - looks like it DOES NOT start a browser, e.g. have the ability to click a button on navigate a single page app. Assuming that's correct, lets not touch the UI at all here and go straight to the API. You may need to just observe what XHRs are being made as you walk through the user flow.

The UI service itself is purely static content, so there's not much point in load testing it. We're effectively load testing Google's static content service.

Reply:
You are correct - it is not in the same category as selenium, for example. Agreed on your suggested approach - that makes a lot of sense.

Reply:
Done.